hr_payslip_change_state/wizard/hr_payslip_change_state.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# © 2016 - Eficent http://www.eficent.com/
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields, api
+from openerp.tools.translate import _
+from openerp.exceptions import Warning as UserError
+
+
+class HrPaySlipChangeState(models.TransientModel):
```

Comment:
Shouldn't it be `HrPayslipChangeState` instead of `HrPaySlipChangeState` (using a lowercase s for payslip)?